event_session_registration_multi_qty/models/event_session.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017-19 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0).
+from odoo import api, models
+
+
+class EventSession(models.Model):
+    _inherit = "event.session"
+
+    @api.depends("registration_ids.qty")
+    def _compute_seats(self):
+        for session in self:
+            if not session.event_id.registration_multi_qty:
+                return super()._compute_seats()
```

Comment:
```suggestion
        records = self.filtered("event_id.registration_multi_qty")
        super(EventSession, self - records)._compute_seats()
        for session in records:
```

Better to call super outside of the loop, otherwise you loose all the recordset optimizations

Reply:
Thanks @ivantodorovich I'll try to get some time and check your comments :)